src/Model/Resolver/ResetPassword.php

```diff
@@ -42,6 +44,13 @@ class ResetPassword implements ResolverInterface {
      */
     protected $customerRepository;
 
+    protected function getAuthentication()
+    {
+        return ObjectManager::getInstance()->get(
+            AuthenticationInterface::class
+        );
+    }
+
```

Comment:
Please add dependencies in the class constructor.
See the example from the constructor of this class.